### ISSUE TITLE
Env with same name is serialized

### ISSUE THREAD

**Issue by jolafrite** [NativeScript/nativescript-dev-webpack#613]
Hi

when using ```tns run ios --bundle --env.aot --env.aot=false```
The aot value will be in the hookArgs available as an array ```[true, false]``` 
But in webpack.config.js it becomes ```true,false```

The problem is coming form this line of code:
https://github.com/NativeScript/nativescript-dev-webpack/blob/a68778e7c576e0ec75b73beedfaa2fee08a97511/lib/compiler.js#L113

Maybe we can change the line with something like:

```js
const args = [];
envFlagNames.map(item => {
    let value = envData[item];
    if(!Array.isArray(value)) {
        value = [value];
    }
    value.map(value => args.push(`--env.${item}=${value}`))
});

return args;
```

**Comment by jolafrite** [comment]
sure

**Comment by jolafrite** [comment]
sent

**Comment by sis0k0** [comment]
10x!

**Comment by sis0k0** [comment]
The issue was fixed with #614 and will be released in the next version of the plugin. Thanks, @jolafrite!
